activities/Clock.activity/js/activity.js

```diff
@@ -16,15 +16,19 @@ define(["sugar-web/activity/activity","sugar-web/env","sugar-web/graphics/radiob
             } else {
                 activity.getDatastoreObject().loadAsText(function(error, metadata, data) {
                     if (error==null && data!=null) {
-                        console.log(data);
-                        clock.changeFace(data[0]);
+                        face=data[0];
+                        if(data[0]=="simple"){
+                            simpleClockButton.onclick();
```

Comment:
I'm not sure it's a good idea to do "onclick" it's easier to do the changeFace like you've done before, then just add the CSS class "active" to the toolbar element.  Something like: document.getElementById("simple-clock-button").classList.add("active");